### PR TITLE
Stories/array

### PR DESCRIPTION
**Pull request by vancuom98php**
Review sương sương giúp e :v

### CODE REVIEW

File: ex02.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<?php
+
+$arr = [];
+
+function enterValue($n, $m)
+{
+    $arr = [];
+    print "<div id='results'>";
+        print "<form class='form-group' method='post'>";
```
Comment (trinhlecong94) [inline ex02.php:9]: should write in html code

File: ex03.php [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        }
+    }
+    return $str;
+}
+
+function findNameInArray($arr, $findName)
+{
+    $str = "";
+    $i = 1;
+    foreach ($arr as $mainKey => $item) {
```
Comment (trinhlecong94) [inline ex03.php:44]: array_search()

File: ex03.php [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+}
+
+function sortArrByScore($arr)
+{
+    $str = "";
+    $i = 1;
+    usort($arr, function ($user1, $user2) {
+        return $user1["Score"] <=> $user2["Score"];
+    });
+    foreach ($arr as $mainKey => $item) {
```
Comment (trinhlecong94) [inline ex03.php:63]: create function print array

File: ex03.php [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+        "Score" => 900
+    ],
+    "User4" => [
+        "Name" => "Lê Văn Cườm",
+        "Age" => 19,
+        "Score" => 990
+    ]
+];
+
+$n = 4;
```
Comment (trinhlecong94) [inline ex03.php:26]: what is this :)))))

Comment (vancuom98php) [reply ex03.php:26]: Originally, I intended to use it for (addArray funnction), but now it's redundant :v

File: ex01.php [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+    return $str;
+}
+
+function findValueInArray($valueFind, $intArr)
+{
+    $str = "";
+    foreach ($intArr as $key => $value) {
+        if ($valueFind == $value) {
+            $str .= $key . " => " . $value;
+            $str .= "\n";
```
[the remark this answers is not in the document]
Comment (vancuom98php) [reply ex01.php:32]: array_search() return only the first matching key

**Pull request merged**